packages/shopware-6-client/src/interfaces/models/content/category/Category.ts

```diff
@@ -0,0 +1,40 @@
+import { Media } from "../media/Media";
+import { Category } from "./Category";
+import { CmsPage } from "../cms/CmsPage";
+import { Product } from "../product/Product";
+import { Entity } from "../../common/Entity";
+import { TagEntity } from "../../system/tag/Tag";
+import { CategoryTranslation } from "./CategoryTranslation";
+import { CustomField } from "../../common/CustomField";
+
+export interface Category extends Entity {
+  parentId: string | null;
+  autoIncrement: number;
+  mediaId: string | null;
+  name: string | null;
+  breadcrumb: string[];
+  level: number;
+  active: boolean;
+  childCount: number;
+  displayNestedProducts: boolean;
+  parent: Category | null;
+  children: Category[] | null;
+  translations: CategoryTranslation[] | null;
+  media: Media | null;
+  products: Product[] | null;
+  nestedProducts: Product[] | null;
+  afterCategoryId: string | null;
+  customFields: CustomField[];
+  tags: TagEntity[] | null;
+  cmsPageId: string | null;
+  cmsPage: CmsPage | null;
+  slotConfig: [] | null;
+  externalLink: string | null;
+  visible: boolean;
+  type: string;
+  description: string;
+  extensions: [];
```

Comment:
missing type

Reply:
Hm, that array comes from very deeply inherited trait, which is `ExtendableTrait.php`. I believe if we go through all the interfaces we would need to add that to most of the interfaces.

I will delete that field for now and we will consider adding that to `ExtendableTrait`, which will be extended by `Entity`, which will be extended by concrete interfaces.

Deleting it for now as we did not include it in all other interfaces.